src/cephadm/cephadm

```diff
@@ -1967,6 +1977,12 @@ def deploy_daemon_units(fsid, uid, gid, daemon_type, daemon_id, c,
             f.write(' '.join(prestart.run_cmd()) + '\n')
         elif daemon_type == CephIscsi.daemon_type:
             f.write(' '.join(CephIscsi.configfs_mount_umount(data_dir, mount=True)) + '\n')
+            ceph_iscsi = CephIscsi.init(fsid, daemon_id)
+            tcmu_container = ceph_iscsi.get_tcmu_runner_container()
+            f.write('! '+ ' '.join(tcmu_container.rm_cmd()) + '\n')
+            if 'podman' in container_path:
+                f.write('! '+ ' '.join(tcmu_container.rm_cmd(storage=True)) + '\n')
+            f.write(' '.join(tcmu_container.run_cmd()) + '\n')
```

Comment:
thought we should make sure this new container is removed like the other container on start up